neurom/apps/tests/test_morph_stats.py

```diff
@@ -125,13 +125,13 @@ def test_eval_stats_applies_numpy_function():
 def test_extract_stats_single_neuron():
     nrn = nm.load_neuron(os.path.join(DATA_PATH, 'Neuron.swc'))
     res = ms.extract_stats(nrn, REF_CONFIG)
-    nt.eq_(res.keys(), REF_OUT.keys())
+    nt.eq_(set(res.keys()), set(REF_OUT.keys()))
```

Comment:
was doing pylint stuff wasn't necessary for tox? it makes the diff hard to read, so I prefer if only one thing is changed at a time

Reply:
ic, good to add that to the commit message so it's known